### PR TITLE
feat(api): Log any access granted as part of a legacy fallback.

### PR DESCRIPTION
**Pull request by ajordens**

### CODE REVIEW

File: fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java [hunk 1]
```diff
@@ -177,6 +177,16 @@ public boolean hasPermission(
     authorizationFailure.set(
         hasPermission ? null : new AuthorizationFailure(a, r, resourceName.toString()));
 
+    if (permission != null && permission.isLegacyFallback() && hasPermission) {
+      // log any access that was granted as part of a legacy fallback.
+      if (a == Authorization.READ) {
```
Comment (robzienert) [inline fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java]: One-letter var names are sadface.

Comment (ajordens) [reply fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java]: indeed they are

Comment (ajordens) [reply fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java]: thanks travis

File: fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java [hunk 2]
```diff
@@ -177,6 +177,16 @@ public boolean hasPermission(
...
+        // purposely logging at 'debug' as 'READ' will be sufficiently more verbose
```
Comment (dreynaud) [inline fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java]: > sufficiently

did you mean significantly?

Comment (ajordens) [reply fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java]: tbh I did mean sufficiently but I speak Canadian English so maybe I'm in the wrong.

**Pull request merged**